Realm/Tests/ArrayPropertyTests.m

```diff
@@ -262,6 +262,60 @@ - (void)testFastEnumeration
                     @"Adding an object during fast enumeration did not throw");
 }
 
+- (void)testValueForKey {
```

Comment:
Please add standalone tests with employees in both `testValueForKey` and `testSetValueForKey`.